pkgs/by-name/ha/harper/package.nix

```diff
@@ -6,16 +6,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "harper";
-  version = "0.15.0";
+  version = "0.16.0";
 
   src = fetchFromGitHub {
     owner = "elijah-potter";
```

Comment:
We should probably change the owner and `meta` to point at the new repository (Elijah moved the repo under the Automattic organization)